Portal/src/Datahub.Portal/Pages/Workspace/FileSearchResult.cs

```diff
@@ -0,0 +1,11 @@
+   namespace Datahub.Portal.Pages.Workspace.Storage
+   {
+       public class FileSearchResult
```

Comment:
Haven't seen the UI and layout but should we include the container and account info maybe?

Reply:
search is per container so this might not be necessary